resources/views/pages/ticket/[ticket].blade.php

```diff
@@ -181,7 +181,7 @@
                 <div class="relative w-full p-2 bg-embed rounded">
                     <x-ticket.stat-element label="Achievement">{!! achievementAvatar($ticket->achievement, iconSize:16) !!}</x-ticket.stat-element>
                     <x-ticket.stat-element label="Game">{!! gameAvatar($ticket->achievement->game, iconSize:16) !!}</x-ticket.stat-element>
-                    <x-ticket.stat-element label="Author">{!! userAvatar($ticket->achievement->author, iconSize:16) !!}</x-ticket.stat-element>
+                    <x-ticket.stat-element label="Author">{!! userAvatar($ticket->achievement->developer->display_name, iconSize:16) !!}</x-ticket.stat-element>
```

Comment:
Same comment about passing developer. Less impact with a singular record.

Reply:
Resolved in latest.